WalletWasabi.Fluent/ViewModels/Wallets/Buy/Messages/UserMessageViewModel.cs

```diff
@@ -46,7 +46,7 @@ private async Task EditAsync(ChatMessage message)
 		Message = newMessage;
 		IsUnread = newMessage.IsUnread;
 		OriginalText = newMessage.Text;
-		UiMessage = newMessage.Text;
+		UiMessage = ParseRawMessage(newMessage);
```

Comment:
I thought the base class will handle this like in other viewmodels. 
Is this a special case?

Reply:
It is, this is `EditAsync()`, this only modifies the text, but doesn't create a new object, so the base constructor isn't called.